pvlib/tests/test_pvsystem.py

```diff
@@ -1689,14 +1689,37 @@ def test_PVSystem_get_irradiance():
                                        irrads['dhi'])
 
     expected = pd.DataFrame(data=np.array(
-        [[ 883.65494055,  745.86141676,  137.79352379,  126.397131  ,
-              11.39639279],
-           [   0.        ,   -0.        ,    0.        ,    0.        ,    0.        ]]),
+        [[883.65494055, 745.86141676, 137.79352379, 126.397131, 11.39639279],
+         [0., -0., 0., 0., 0.]]),
                             columns=['poa_global', 'poa_direct',
                                      'poa_diffuse', 'poa_sky_diffuse',
                                      'poa_ground_diffuse'],
                             index=times)
+    assert_frame_equal(irradiance, expected, check_less_precise=2)
+
 
+def test_PVSystem_get_irradiance_albedo():
+    system = pvsystem.PVSystem(surface_tilt=32, surface_azimuth=135)
+    times = pd.date_range(start='20160101 1200-0700',
+                          end='20160101 1800-0700', freq='6H')
+    location = Location(latitude=32, longitude=-111)
+    solar_position = location.get_solarposition(times)
```

Comment:
I see the solar position calculation is copied from the test above, but I'd rather see them hard coded (doesn't need to be precise!) or refactored into a fixture. Not a blocker.